### PR TITLE
Server Skeleton Code

### PR DESCRIPTION
**Pull request by adibb**
The Flask code scummed from old projects, cleaned up to be skeletal and ready for others to build off of.

### CODE REVIEW

File: SERVER/targets.d/CONFIG.skel.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""
+Configuration of syllabus server.
+Edit to fit development or deployment environment.
+
+"""
+
+PORT=8080
+DEBUG = True  # Set to False for production use
+secret_key = "fmavKUW0Mwmc0uVLJPx84OLEpS6taGMBS5e1LwZ5lKUOItRZvFaVPTW0B6rEJVhJYdPNPs7Npw5gH0IG"
```
Comment (zachary-trudo) [inline SERVER/targets.d/CONFIG.skel.py]: Traditionally - secrets shouldn't be in a repo... but I'm not too concerned here.

File: SERVER/targets.d/choose_config.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#
+# 
+
+# What machine is this?  Use uname to find hardware
+# 
+architecture=`uname -m`
+node=`uname -n`
+processor=`uname -p`
+opsys=`uname -v`
+port="8080"   # But see how this is altered for shared server ix
```
Comment (zachary-trudo) [inline SERVER/targets.d/choose_config.sh:18]: Can we use multiple ports? or just one?

File: SERVER/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (zachary-trudo) [inline SERVER/templates/index.html:1]: Can these be generated dynamically with flask?

**Review by zachary-trudo — APPROVED** [review]
Overall looks good... I've a few questions. If you could answer them, I'll merge.

File: SERVER/targets.d/CONFIG.skel.py (hunk 1 above)
Comment (adibb) [reply SERVER/targets.d/CONFIG.skel.py]: D'oh! I'll change it to a string that states as much and note it in the README.

Comment (adibb) [reply SERVER/targets.d/CONFIG.skel.py]: Revealed a couple of dead files and comments that weren't correct anymore in the autoconfig. Tweaking them to match.

File: SERVER/targets.d/choose_config.sh (hunk 2 above)
Comment (adibb) [reply SERVER/targets.d/choose_config.sh:18]: Not hard to add another port to the configuration script. More difficult to convince Flask to listen on two at once. Need to look it up.

File: SERVER/templates/index.html (hunk 3 above)
Comment (adibb) [reply SERVER/templates/index.html:1]: The HTML files themselves? Not to my knowledge, though Flask isn't something I've explored a whole bunch. I know they include Jinja2 templates, so you can do stuff like 
```
<!doctype html>
<title>Hello from Flask</title>
{% if name %}
  <h1>Hello {{ name }}!</h1>
{% else %}
  <h1>Hello, World!</h1>
{% endif %}`
```
but I'm not sure that's what you're asking.

Comment (zachary-trudo) [reply SERVER/templates/index.html:1]: Mmk not a big deal at this point. Thanks for looking! Jinja2 templates aren't bad although sometimes I get irritated working with them.

File: SERVER/targets.d/choose_config.sh (hunk 2 above)
Comment (zachary-trudo) [reply SERVER/targets.d/choose_config.sh:18]: Ok cool. Thanks

File: SERVER/targets.d/CONFIG.skel.py (hunk 1 above)
Comment (zachary-trudo) [reply SERVER/targets.d/CONFIG.skel.py]: Ok  - I'll hold off merging until you've squashed your changes

**Pull request merged**